app/templates/components/ui-table/cell/events/view/sessions/cell-buttons.hbs

```diff
@@ -1,4 +1,4 @@
-<UiDropdown @class="ui floating dropdown button">
+<UiDropdown @class="ui floating dropdown button max-content-button">
```

Comment:
?

Reply:
@iamareebjamal yes sir,  I am doing this,  as committed your suggested changes from my phone,   I am changing it asap.